SPECS/llvm/llvm.spec

```diff
@@ -37,6 +37,7 @@ export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
 mkdir -p build
 cd build
 cmake -G Ninja                              \
+      -DCMAKE_CXX_FLAGS="-fno-strict-aliasing" \
```

Comment:
Instead of disabling strict aliasing for the whole build, could we instead use [the patch](https://github.com/NixOS/nixpkgs/pull/152321) mentioned in the [49955 issue](https://github.com/llvm/llvm-project/issues/49955) you referenced?

Reply:
If you're comfortable with the minimalist approach I can include the same patch in this commit and apply it instead of what I'm currently doing, you're right that it appears to be sufficient though I have no idea why.